src/routes/comment.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const router = express.Router();
+
+const commentController = require('../controllers/comment');
+const postController = require('../controllers/comment');
+
+router.post('/:id', postController.addComment);
+router.get('/:postid', commentController.getOneComment);
+router.delete('/:postid', commentController.removeComment);
```

Comment:
Since these routes will be prefixed by `/api/comments` it doesn't make sense to put `postid` in the route.
You can change to `/:commentid` and put `postid` as a query param so that the full route becomes `/api/comments/:commentid?post=:postid`. This structure will make more logical sense.